tests/test_user_roles/test_query_user_role.py

```diff
@@ -34,36 +35,37 @@ def test_query_users_role(self):
         expected_responese = user_role_query_response
         self.assertEqual(execute_query, expected_responese)
 
-    def test_query_users_role_by_role(self):
-        user = User(email='mrm@andela.com', location="Lagos",
-                    name="test test",
-                    picture="www.andela.com/testuser")
-        user.save()
-        user_role = UsersRole(user_id=user.id, role_id=1)
-        user_role.save()
-        db_session().commit()
+    # def test_query_users_role_by_role(self):
```

Comment:
hello, I'd love to know why these tests commented out?

Reply:
+1

Reply:
This test was uncommented and refactored to pass. thanx @C3real-kill3r for pointing it out.